Dockerfile

```diff
@@ -15,6 +15,7 @@ ENV VERNEMQ_VERSION 1.3.1
 # Defaults
 ENV DOCKER_VERNEMQ_KUBERNETES_NAMESPACE default
 ENV DOCKER_VERNEMQ_KUBERNETES_APP_LABEL vernemq
+ENV DOCKER_VERNEMQ_LOG.CONSOLE console
```

Comment:
Shouldn't this be `DOCKER_VERNEMQ_LOG__CONSOLE` to ensure that the K8s folk get the same experience out of the box?

Reply:
Ahh very true, missed that part in the docs, haven't used k8s too much myself

Reply:
Done!